source/standalone/workflows/skrl/play.py

```diff
@@ -26,6 +26,14 @@
 parser.add_argument("--num_envs", type=int, default=None, help="Number of environments to simulate.")
 parser.add_argument("--task", type=str, default=None, help="Name of the task.")
 parser.add_argument("--checkpoint", type=str, default=None, help="Path to model checkpoint.")
+parser.add_argument(
+    "--ml_framework",
+    type=str,
+    default="torch",
+    choices=["torch", "jax", "jax-numpy"],
```

Comment:
Above in the code, you only mentioned "jax" and "torch". If "jax-numpy" is also supported, please fill all the doc strings.